archetypes/mule-app/src/main/resources/archetype-resources/src/test/java/ExampleFunctionalTestCase.java

```diff
@@ -1,34 +1,45 @@
 /**
- * This file was automatically generated by the Mule Development Kit
+ * This file was automatically generated by the Mule ESB Maven Tools
  */
-
 #set( $symbol_pound = '#' )
 #set( $symbol_dollar = '$' )
 #set( $symbol_escape = '\' )
-
 package ${package};
 
-import org.mule.api.MuleMessage;
-import org.mule.api.client.MuleClient;
-import org.mule.transport.NullPayload;
-import org.junit.Test;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
+import org.junit.Test;
+import org.mule.DefaultMuleMessage;
+import org.mule.api.MuleMessage;
+import org.mule.api.client.MuleClient;
+import org.mule.tck.junit4.FunctionalTestCase;
+import org.mule.transport.NullPayload;
 
-public class FunctionalTestCase extends org.mule.tck.junit4.FunctionalTestCase
+public class ExampleFunctionalTestCase extends FunctionalTestCase
 {
-    protected String getConfigResources()
+    @Override
+    protected String getConfigFile()
+    {
+        return "mule-config.xml";
+    }
+
+#if(${domainGroupId} != 'empty')
```

Comment:
Actually, this will only be available from certain versions. So we need to figure out what to do here.